include/awkward/array/RawArray.h

```diff
@@ -1292,9 +1292,9 @@ namespace awkward {
   private:
     const kernel::lib ptr_lib_;
     const std::shared_ptr<T> ptr_;
-    const int64_t offset_;
-    const int64_t length_;
-    const int64_t itemsize_;
+    const int64_t offset_ = 0;
+    const int64_t length_ = 0;
+    const int64_t itemsize_ = 0;
```

Comment:
Same question as above on `IndexOf<T>`: if there's any way for `offset_`, `length_`, or `itemsize_` to be uninitialized in a `RawArrayOf<T>` instance, then that itself is a bug. We don't want to hide it by giving these fields default values.